DEVELOPMENT.md

```diff
@@ -162,8 +162,36 @@
     )
     ```
 
-4.  Build the new target.
+4. Point to local gapic-generator-java
```

Comment:
Thanks Tomo! It's good to have a section that points to local repo, just be aware that this section will change very soon once we merge https://github.com/googleapis/gapic-generator-java/pull/918. We will point googleapis to a local jar instead of a path to local repo.

Reply:
That sounds good. Not a blocker for this pull request.